src/bin/script-parsers.test.js

```diff
@@ -12,6 +12,47 @@ describe('webpack config name parser', () => {
       webpackConfig('webpack --profile --json > stats.json') === 'webpack.config.js'
     )
   })
+
+  it('should return undefined when script does not call the webpack cli', () => {
+    const sciptsThatDoNotCallWebpackCli = [
+      'karma start',
+      'istanbul check-coverage --statements 23 --branches 5 --functions 9 --lines 24',
+      'npm test -- --auto-watch --no-single-run',
+      'npm-run-all --parallel validate-webpack:* lint test --serial check-coverage',
+      'webpack-validator webpack.config.js --env.dev',
+      'webpack-validator webpack.config.js --env.prod',
+      'rimraf dist',
+      'cpy src/index.html src/favicon.ico dist',
+      'npm run clean-dist && npm run copy-files',
+      'npm run clean-and-copy',
+      'webpack-dev-server --env.dev --content-base dist',
+      'npm run clean-and-copy',
+      'npm run clean-and-copy',
```

Comment:
My bad, I had just copied the scripts from:
https://github.com/kentcdodds/es6-todomvc/blob/c7838ebff426c15bdc16344863bacda11e175508/package.json

I didn't realise that `prebuild` and `prebuild:prod` were the same